translator/internaldata/metrics_to_oc_test.go

```diff
@@ -87,13 +87,7 @@ func TestMetricsToOC(t *testing.T) {
 		},
 
 		{
-			name:     "all-types-no-data-points",
```

Comment:
Should this test have been removed?

Reply:
Cannot make it pass for the moment since no diff between int/double when no points.